evopy/evopy.py

```diff
@@ -54,11 +59,18 @@ def run(self):
         return best.genotype
 
     def _init_population(self):
+        if self.strategy == Strategy.SINGLE_VARIANCE:
+            strategy_parameters = [np.random.randn()]
+        elif self.strategy == Strategy.MULTIPLE_VARIANCE:
+            strategy_parameters = np.random.randn(self.individual_length)
+        else:
+            strategy_parameters = np.random.randn(
```

Comment:
If this is indeed the full variance, it does not seem right to me, we should have `self.individual_length` parameters for the sigmas and indeed `(self.individual_length + 1) * self.individual_length / 2` parameters for the rotation angles. So we are missing the sigmas here.